framework/behaviors/SluggableBehavior.php

```diff
@@ -239,7 +241,7 @@ protected function validateSlug($slug)
     /**
      * Generates slug using configured callback or increment of iteration.
      * @param string $baseSlug base slug value
-     * @param int $iteration iteration number
+     * @param integer $iteration iteration number
```

Comment:
Thanks! These comment adjustments have been made automatically by Phpstorm when using the autoformatter. Will revert them and add unit tests tomorrow.